run_brer/plugin_configs.py

```diff
@@ -150,6 +150,8 @@ def build_plugin(self):
         WorkElement = _get_workelement()
         if self.get_missing_keys():
             raise KeyError('Must define {}'.format(self.get_missing_keys()))
+        if self.get('alpha') == 0:
```

Comment:
Nit picking, but this is a floating point value, so we should compare it to a value of the same type.
```suggestion
        if self.get('alpha') == 0.0:
```

(Note: it is often problematic to try to compare floating point numbers with exact equality because of the varying precision of binary floating point representations, but 0.0 and 1.0 (and reasonably sized powers of 2) are always exact.)